### PR TITLE
Update zeppelin example for DCOS 1.11

### PR DESCRIPTION
**Pull request by josepedro**
- Updating prerequisites;
- Updating Zeppelin version;
- Updating DC/OS UI services;
- Updating Zeppelin UI;
- Updating Zeppelin example;
- Updating Zeppelin tutorial;
- Updating Zeppelin urls documentation.

### CODE REVIEW

File: zeppelin/1.11/README.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+- [Prerequisites](#prerequisites)
+- [Install Zeppelin](#install-zeppelin)
+- [Use Zeppelin](#use-zeppelin)
+- [Uninstall Zeppelin](#uninstall-zeppelin)
+
+## Prerequisites
+
+- A running DC/OS 1.11 cluster with 2 agents (one private, one public) each with 2 CPUs and 2 GB of RAM available.
+- [DC/OS CLI](https://dcos.io/docs/1.9/usage/cli/install/) installed.
```
Comment (pleia2) [inline zeppelin/1.11/README.md]: Should be updated to link to the 1.11 docs: https://docs.mesosphere.com/1.11/cli/install/

File: zeppelin/1.11/README.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+- [Marathon-LB](https://dcos.io/docs/1.9/usage/service-discovery/marathon-lb/) installed.
```
Comment (pleia2) [inline zeppelin/1.11/README.md]: Same here, for 1.11 this should point to the service docs: https://docs.mesosphere.com/services/marathon-lb/

File: zeppelin/1.11/README.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+
+## Install Zeppelin
+
+You typically want to access Zeppelin via a web browser outside of the DC/OS cluster. To access the Zeppelin UI from outside of the DC/OS cluster you can use [Marathon-LB](https://dcos.io/docs/1.9/usage/service-discovery/marathon-lb/), which is recommended for production usage. The following steps will install zeppelin:
```
Comment (pleia2) [inline zeppelin/1.11/README.md]: Since it's linked above, you can remove this old link to 1.9.

**Comment by josepedro** [comment]
done, thank you so much @pleia2

**Pull request merged**